lib/action.js

```diff
@@ -108,6 +108,15 @@ Action.prototype.getName = function () {
 };
 
 /**
+ * Action#isRequired -> Boolean
+ *
+ * Return true if required
+ **/
+Action.prototype.isRequired = function () {
```

Comment:
I see something similar in mainstream https://github.com/python/cpython/blob/2bd37c227e8a042e036c7455d974e3d0b36aed53/Lib/argparse.py#L817.

Seems ok.